common/setups/rasr/nn_system.py

```diff
@@ -454,7 +454,7 @@ def nn_recognition(
         self,
         name: str,
         returnn_config: returnn.ReturnnConfig,
-        checkpoints: List[returnn.Checkpoint],
+        returnn_models: List[returnn.ReturnnModel],
```

Comment:
Why did you change the Checkpoints back to Model? Is there something the Checkpoint can not do? Because I remember that we discussed at some point that ReturnnModel should be deprecated because it is not supported by some jobs (e.g. CheckpointAveraging or GetBestCheckpoint).

Reply:
No, I wrote that down incorrectly. I ll revert back then

Reply:
But why was that incorrectly? I am using `Checkpoint` for my CTCSystem and this seems to be more consistent with the rest of the setups.

Reply:
I wrote down that Checkpoint was deprecated. I didn't recheck that. My mistake

Reply:
That was incorrect of me